helm/idol-nifi/templates/zookeeper/zookeeper.yaml

```diff
@@ -60,7 +63,7 @@ spec:
             value: "server.1=$(POD_IP):2888:3888;2181"
         envFrom:
           - configMapRef:
-              name: zookeeper-cm
+              name: {{ $nifiCluster.clusterId }}-zk-cm
```

Comment:
Is there any need for a separate ConfigMap per cluster if they're all the same data?